validator/keymanager/remote-web3signer/v1/requests.go

```diff
@@ -263,3 +271,65 @@ func GetSyncCommitteeContributionAndProofSignRequest(request *validatorpb.SignRe
 		ContributionAndProof: contribution,
 	}, nil
 }
+
+// GetBlockV2BellatrixSignRequest maps the request for signing type BLOCK_V2 BELLATRIX.
+func GetBlockV2BellatrixSignRequest(request *validatorpb.SignRequest, genesisValidatorsRoot []byte) (*BlockV2BellatrixSignRequest, error) {
+	if request == nil {
+		return nil, errors.New("nil sign request provided")
+	}
+	var b interfaces.BeaconBlock
+	switch request.Object.(type) {
+	case *validatorpb.SignRequest_BlindedBlockV3:
+		BlindedBlockV3, ok := request.Object.(*validatorpb.SignRequest_BlindedBlockV3)
+		if !ok {
+			return nil, errors.New("failed to cast request object to blinded block v3")
+		}
+		if BlindedBlockV3 == nil {
+			return nil, errors.New("invalid sign request: BlindedBlockV3 is nil")
```

Comment:
Colons separate different errors, using them within errors may cause unwanted parsing. You can replace the colon with a `-` perhaps?